pkg/virt-controller/watch/network.go

```diff
@@ -91,19 +93,37 @@ func trimDoneInterfaceRequests(vm *v1.VirtualMachine, vmi *v1.VirtualMachineInst
 	vm.Status.InterfaceRequests = updateIfaceRequests
 }
 
-func handleDynamicInterfaceRequests(vm *v1.VirtualMachine) {
+func handleDynamicInterfaceRequests(vm *v1.VirtualMachine, vmiHasPodNetwork bool, clusterConfig *virtconfig.ClusterConfig) error {
 	if len(vm.Status.InterfaceRequests) == 0 {
-		return
+		return nil
 	}
 
 	vmTemplateCopy := vm.Spec.Template.Spec.DeepCopy()
+
+	// Explicitly add the default interface before hot-plugging another interface
```

Comment:
nit: I would change the note to explain why the following is necessary, which is the fact that Kubevirt adds an interface by default to VMIs of a VMs defined with no interfaces.